src/Lounge.Web/Pages/Stats.cshtml

```diff
@@ -0,0 +1,640 @@
+@page
+@using Lounge.Web.Settings
+@using Microsoft.AspNetCore.Mvc.Localization
+@inject ILoungeSettingsService LoungeSettingsService
+@inject IViewLocalizer Localizer
+
+@model Lounge.Web.Pages.StatsPageModel
+@{
+    ViewData["Title"] = $"{Localizer.GetString("Stats")} - {Localizer.GetString("Season")} {Model.Season}";
+    Layout = "_Layout";
+}
+
+<div class="navbar navbar-expand-sm navbar-dark bg-dark rounded py-0" style="margin-bottom: 14px;">
+    <div class="container-fluid">
+        <span class="navbar-brand">@Localizer["Seasons"]</span>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#seasonSelector">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse justify-content-md-center" id="seasonSelector">
+            <ul class="navbar-nav">
+                @foreach (var season in LoungeSettingsService.ValidSeasons)
+                {
+                    <li class="nav-item">
+                        <a class="nav-link @(season == Model.Season ? "active" : "")" asp-page="Stats" asp-route-season="@season">@Localizer["Season"] @season</a>
+                    </li>
+                }
+            </ul>
+        </div>
+    </div>
+</div>
+
+<input type="hidden" value="@Model.Season" id="seasonInput"/>
+
+<h1 class="m-4 text-center">Division Data</h1>
+
+<canvas id="statDivisionChartBody"></canvas>
+
+<div class="row m-2">
+    <div class="table-responsive table-sm col-lg-6 col-md-6 col-sm-12 col-xs-12" style="font-size: 12px; display: inline-block">
+        <table id="stats-table" class="table table-striped table-dark text-center">
+            <thead>
+                <tr>
+                    <th>@Localizer["Division"]</th>
```

Comment:
We should create resx files with these localizer keys for each language, by default for missing it just uses the string though so technically it works, but I also want to have the resx files for if someone does contribute translations.

Reply:
5c0f948b800e041d2dfbc4a3b9a63caab31c2401